hal/Cargo.toml

```diff
@@ -1,227 +1,342 @@
+#===============================================================================
+# Package data
+#===============================================================================
+
 [package]
-name = "atsamd-hal"
-version = "0.15.1"
 authors = [
-    "Wez Furlong <wez@wezfurlong.org>",
-    "Paul Sajna <sajattack@gmail.com>",
-    "Michael van Niekerk <mvniekerk@gmail.com>",
-    "Jesse Braham <jesse@beta7.io>",
+  "Wez Furlong <wez@wezfurlong.org>",
+  "Paul Sajna <sajattack@gmail.com>",
+  "Michael van Niekerk <mvniekerk@gmail.com>",
+  "Jesse Braham <jesse@beta7.io>",
+  "Bradley Harden <bradleyharden@gmail.com>",
 ]
+categories = ["embedded", "hardware-support", "no-std"]
 description = "HAL and Peripheral access API for ATSAMD11, ATSAMD21, ATSAMD51, ATSAME51, ATSAME53 and ATSAME54 microcontrollers"
+documentation = "https://docs.rs/crate/atsamd-hal/"
```

Comment:
should this point to github.io instead?

Reply:
My first instinct is no, because the github.io docs are built from `master`, which is not on crates.io. But as you said yesterday, we should find a way to make those docs more visible.

I guess the bigger question is which set of docs should new users encounter first? A version tied to a specific crates.io release? Or a version built directly from `master`?